arch/arm/src/samv7/sam_lowputc.c

```diff
@@ -327,14 +327,10 @@ void sam_lowsetup(void)
    */
 
 #  if defined(CONFIG_SAMV7_JTAG_FULL_ENABLE)
-#    warning CONFIG_SAMV7_JTAG_FULL_ENABLE is incompatible with CONFIG_SAMV7_USART1.
-#    warning The SYSIO Pin4 must be bound to PB4 to use USART1
+#    error CONFIG_SAMV7_JTAG_FULL_ENABLE is incompatible with CONFIG_SAMV7_USART1.
```

Comment:
This should now be redundant with the same check in `sam_gpio.c`, no?

Reply:
Ok. I can remove one from here